source/agora/node/FullNode.d

```diff
@@ -466,4 +486,71 @@ public class FullNode : API
     protected void onAcceptedTransaction () @safe
     {
     }
+
+    /***************************************************************************
+
+        Push the block data to the `block_handlers` target server list
+        set in config.
+
+        Convert block data to JSON-type string and send it POST using Rest.
+
+        Params:
+            block = externalized block data
+
+    ***************************************************************************/
+
+    protected void pushBlock (const ref Block block) @safe
+    {
+        foreach (BlockExternalizedHandler handler; block_handlers)
+        {
+            runTask({
+                auto jsonStr = block.serializeToJsonString();
```

Comment:
This is doing manual work which the vibe.d serializer already does for you under the hood.

And secondly, using a map type was unnecessary.